pkg/kubelet/apis/deviceplugin/v1alpha1/constants.go

```diff
@@ -30,4 +30,19 @@ const (
 	DevicePluginPath = "/var/lib/kubelet/device-plugins/"
 	// KubeletSocket is the path of the Kubelet registry socket
 	KubeletSocket = DevicePluginPath + "kubelet.sock"
+
+	// ErrFailedToDialDevicePlugin is the error raised when the device plugin could not be
+	// reached on the registered socket
+	ErrFailedToDialDevicePlugin = "Failed to dial device plugin:"
```

Comment:
Please move these global error out of the API

Reply:
done.